client/client.go

```diff
@@ -246,8 +250,10 @@ func IsTwoFactorError(err error) bool {
 
 // SysInfo holds system information
 type SysInfo struct {
-	Series  string `json:"series,omitempty"`
-	Version string `json:"version,omitempty"`
+	Series    string `json:"series,omitempty"`
+	Version   string `json:"version,omitempty"`
+	OS        string `json:"os-id,omitempty"`
```

Comment:
Instead of having this, let's please decorate release.OS with proper json tags, and use it as this field's type, so it becomes a sub-document.